frontend/src/cloudformation_output.json

```diff
@@ -1,9 +1,9 @@
 {
     "Stacks": [
         {
-            "StackId": "arn:aws:cloudformation:eu-west-2:994541973446:stack/diagrams/406c2860-8ac2-11ea-9ac5-060e4e394d84",
+            "StackId": "arn:aws:cloudformation:eu-west-2:994541973446:stack/diagrams/596bc8e0-8b97-11ea-af37-021ebf3a767c",
             "StackName": "diagrams",
-            "ChangeSetId": "arn:aws:cloudformation:eu-west-2:994541973446:changeSet/samcli-deploy1588245350/19d5d496-7d90-4936-8dcf-65552ab8156f",
+            "ChangeSetId": "arn:aws:cloudformation:eu-west-2:994541973446:changeSet/samcli-deploy1588344666/122655d2-21e0-4f36-b025-114a74a294d2",
```

Comment:
I think this page might be suggesting that this is sensitive info: https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/using-cfn-updating-stacks-changesets-create.html 

All that we care about is the outputs... perhaps we should just remove everything else?